prime.js

```diff
@@ -0,0 +1,29 @@
+function assert(bool, message) {

```

Comment:
we better name the param as `condition` than the `bool`

Reply:
Thank you for your feedback. I have addressed the requested changes in the latest commit.